### ISSUE TITLE
CascadeInputTransparent = false not work

### ISSUE THREAD

**Issue by xtuzy** [dotnet/maui#10040]
### Description

I need a view in scrollview not scroll when pan in it, so i try set

```
scrollView.InputTransparent = true;
scrollView.CascadeInputTransparent = false;
```
but after set this, no pan event be invoked.

### Steps to Reproduce

1. clone my project and run

### Link to public reproduction project repository

https://github.com/xtuzy/MauiAppInputTransparentBug

### Version with bug

6.0.486 (current)

### Last version that worked well

Unknown/Other

### Affected platforms

Android, Windows, I was *not* able test on other platforms

### Affected platform versions

Android 9

**Comment by xtuzy** [comment]
Windows


https://user-images.githubusercontent.com/17793881/189520641-b6702410-14a5-4ed3-99a9-ba47876d6dc7.mp4


Android


https://user-images.githubusercontent.com/17793881/189520645-eb34454c-3f5c-4ed1-9806-01424dd0b2ea.mp4

**Comment by mattleibow** [comment]
I believe this is actually not related to the input transparency, but the fact that the gesture recognizers never actually end, thus never re-enabling the input. I tested this by putting a random timer that set `InputTransparent=false` and scrolling started again.

However, this is not 100% the cause - it is just the first one. On .NET 7.0.59, the scroll view seems to keep scrolling even when the gesture is running and transparency is enabled.

I created this issue to track the gestures issue: https://github.com/dotnet/maui/issues/14869
